### ISSUE TITLE
warnings on compile 

### ISSUE THREAD

**Issue by Cncninja** [grblHAL/LPC176x#7]
Are this anything to worry about?

![ice_screenshot_20210906-235015](https://user-images.githubusercontent.com/75448047/132266350-cbfe4c1a-1d27-4aac-9ca0-0cc52a8b7d36.jpeg)

![ice_screenshot_20210906-234957](https://user-images.githubusercontent.com/75448047/132266354-e47e24f7-795d-4459-a6ce-d2a4aa7b0d20.jpeg)

**Comment by terjeio** [comment]
No, I just ignore these warnings. It is due to some bad code from NXP, perhaps one should take a look at why they are issued...

**Comment by Cncninja** [comment]
thank you

**Comment by Cncninja** [comment]
The 2130 still has errors with a fresh download as well as the SD card plug-in. i deleted the 2130 and re-commented out the SD card and it compiled. I don't use 2130s or the SD, so no issue for me but just letting you know in case the plugins are affecting other builds. 
Thank you, do you have a donation link?

**Comment by terjeio** [comment]
> The 2130 still has errors 

Which errors? It compiles without errors for me with both 2130 and SD card enabled.
The iap warnings are on NXP - not my code.

> do you have a donation link?

Sorry, I do not.

**Comment by Cncninja** [comment]
![ice_screenshot_20210921-072359](https://user-images.githubusercontent.com/75448047/134189173-2c97ce27-07d4-4778-abd7-1ed0584de295.jpeg)
![ice_screenshot_20210921-072348](https://user-images.githubusercontent.com/75448047/134189226-8d74ec40-769b-499c-b926-9f6c4dddf939.jpeg)
I deleted the 2130 folders so i don't see those errors but this is the SD card error, I don't use the SD so no problem for me but not sure if this would affect other builds.
Thank you for your work.

**Comment by terjeio** [comment]
Ok, if you uncomment SDCARD_ENABLE you have to [fetch the FatFs source](https://github.com/grblHAL/LPC176x/tree/master/src/FatFs), I'll add it to the next commit so that downloading will no longer be required .
